ceremonies/src/lib.rs

```diff
@@ -526,13 +528,13 @@ impl<T: Config> Module<T> {
         let cids = <encointer_communities::Module<T>>::community_identifiers();
         for cid in cids.iter() {
             let cindex = <encointer_scheduler::Module<T>>::current_ceremony_index() - 1;
-            let meetup_count = Self::meetup_count((cid, cindex));
             let reward = Self::nominal_income(cid);
 
-            for m in 1..=meetup_count {
+            for m in MeetupLocationIndex::<T>::iter_prefix_values((cid, cindex))
+                .collect::<BTreeSet<MeetupLocationIndexType>>().iter() {
                 // first, evaluate votes on how many participants showed up
```

Comment:
Do we need to collect here and turn it into an iterator again? I believe we can do better.

Reply:
the collection into a BTreeSet removes all duplicates